tests/test_config.py

```diff
@@ -187,6 +187,149 @@ def test_get_pull_registries(self, config, expected):
 
         assert pull_registries == expected
 
+    @pytest.mark.parametrize(('config', 'expected_slots_dir', 'expected_enabled_hosts'), [
+        ("""\
```

Comment:
`textwrap.dedent` could probably be helpful to organize the following text blocks.

Reply:
I'll probably leave it as it is. We can file a technical improvement story if need it and adjust blocks in the whole file.